packages/datagrid/src/components/DefaultHeaderRenderer/MenuDropDownQualityBar.component.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Overlay, Popover, Button } from 'react-bootstrap';
+import { Action } from '@talend/react-components';
+import './MenuDropDownQualityBar.scss';
+
+
+export default function MenuDropDownQualityBar(props) {
```

Comment:
Could we spread the props in the function signature in order to have better visibility of the parameters ?